authentication/views/authentication.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+from django.conf import settings
+import jwt
+from rest_framework import status, views
+from rest_framework.exceptions import AuthenticationFailed
+from rest_framework.response import Response
+
+from authentication.token import decode_token
+
+
+class LoginView(views.APIView):
+    """This view takes a JWT as a GET parameter, verifies it and sets it as a cookie."""
+
+    authentication_classes = ()
+    permission_classes = ()
+
+    def get(self, request, format=None, *args, **kwargs):
+        if settings.JWT_AUTH_SETTINGS["GET_PARAMETER"] in request.GET:
+            # Retrieve the token.
+            token = request.GET.get(settings.JWT_AUTH_SETTINGS["GET_PARAMETER"])
```

Comment:
You can use a `[]` syntax instead of `get` since you know the keys is in `request.GET`. Another way could be 
```suggestion
            token = request.GET.get(settings.JWT_AUTH_SETTINGS["GET_PARAMETER"], None)
            if token:                
```